code/modules/cargo/markets/market_uplink.dm

```diff
@@ -157,6 +157,9 @@
 	accessible_markets = list(/datum/market/blackmarket)
 	custom_premium_price = PAYCHECK_CREW * 2.5
 
+/obj/item/market_uplink/blackmarket/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_CONTRABAND, TRAIT_GENERIC)
```

Comment:
These can be made in-round and pretty easily too. This may actually be the most reasonable method to complete contraband bounties with relatively little difficulty. I know it is a contraband item in of itself, but I figured I would point that out.